packages/devtools_app/lib/src/config_specific/flutter/framework_initialize/_framework_initialize_web.dart

```diff
@@ -25,6 +25,23 @@ Future<String> initializePlatform() async {
     setGlobal(Storage, BrowserStorage());
   }
 
+  // Prevent the browser default behavior for specific keybindings we'll later
+  // handle in the app. This is a workaround for
+  // https://github.com/flutter/flutter/issues/58119.
+  window.onKeyDown.listen((event) {
+    if (!event.metaKey) {
+      return;
+    }
+
+    if (!window.navigator.userAgent.contains('Macintosh')) {
```

Comment:
maybe comment on why the default is only prevented on mac.

Reply:
Will do -